src/Microsoft.AspNetCore.Mvc.Formatters.Xml/Internal/FormattingUtilities.cs

```diff
@@ -15,13 +12,6 @@ public static class FormattingUtilities
     {
         public static readonly int DefaultMaxDepth = 32;
 
-#if NET46
-        public static readonly XsdDataContractExporter XsdDataContractExporter = new XsdDataContractExporter();
```

Comment:
Bit surprised `XsdDataContractExporter` isn't included in .NET Core App. Thought the XML assemblies were the subject of a long .NET Standard discussion and that the decision was to included almost everything.